front/app/containers/Admin/messaging/CustomEmails/All/index.tsx

```diff
@@ -41,17 +43,21 @@ const NoCampaignsDescription = styled.p`
   max-width: 450px;
 `;
 
-interface DataProps extends GetCampaignsChildProps {}
+const CustomEmails = () => {
+  const location = useLocation();
+  const query = parse(location.search, { ignoreQueryPrefix: true });
+  const pageNumber = query.pageNumber;
+  const pageNumberInt = pageNumber ? parseInt(pageNumber as string, 10) : 1;
```

Comment:
If you search the codebase for `const pageNumber`, it looks like we do it differently in most places. Is it possible to stick to the same method here?

Reply:
Otherwise, if you choose to stick with the current implementation, please add an item to the FE CDM, so we can look at streamlining this.

Reply:
this was removed with the new pagination setup